audit/projects/kubernetes-public/buckets/k8s-metrics/iam.json

```diff
@@ -17,15 +22,14 @@
     },
     {
       "members": [
-        "serviceAccount:k8s-metrics@k8s-infra-prow-build-trusted.iam.gserviceaccount.com"
+        "serviceAccount:k8s-triage@k8s-infra-prow-build-trusted.iam.gserviceaccount.com"
       ],
       "role": "roles/storage.legacyBucketWriter"
     },
     {
       "members": [
-        "group:k8s-infra-artifact-admins@kubernetes.io",
         "group:k8s-infra-prow-oncall@kubernetes.io",
-        "serviceAccount:k8s-metrics@k8s-infra-prow-build-trusted.iam.gserviceaccount.com"
+        "serviceAccount:k8s-triage@k8s-infra-prow-build-trusted.iam.gserviceaccount.com"
```

Comment:
This would explain why https://testgrid.k8s.io/wg-k8s-infra-prow#metrics-bigquery is failing

I must have made a copy paste error with https://github.com/kubernetes/k8s.io/pull/2461, will open a fixup PR shortly

Reply:
Fixup PR https://github.com/kubernetes/k8s.io/pull/2548